devtools/platform-descriptor-json/src/main/resources/templates/dockerfile-native.ftl

```diff
@@ -16,13 +16,7 @@
 ###
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
 WORKDIR /work/
-COPY ${build_dir}/*-runner /work/application
-
-# set up permissions for user `1001`
-RUN chmod 775 /work /work/application \
-  && chown -R 1001 /work \
-  && chmod -R "g+rwX" /work \
-  && chown -R 1001:root /work
+COPY --chown=1001:root ${build_dir}/*-runner /work/application
```

Comment:
just a quick comment: we lost the `chmod` in passing. It's not a problem?

/cc @cescoffier

Reply:
@gsmet I see, but not a problem in my tests.

Reply:
@gsmet Don't know if podman or other docker versions have problems.
Also my Windows docker deamon does it correct.

> SECURITY WARNING: You are building a Docker image from Windows against a non-Windows Docker host. All files and directories added to build context will have '-rwxr-xr-x' permissions. It is recommended to double check and reset permissions for sensitive files and directories.

Reply:
Well, you're the one who insisted on it :)

https://github.com/quarkusio/quarkus/pull/6461

Reply:
Yes, seams my newer docker daemon fixed this already.

Reply:
@cescoffier IIRC you added these commands, mind taking a look?

Reply:
These commands were the recommendation made by Ope3nshift and Podman. 

The change looks good but it needs to be tested carefully.

Reply:
Do you have automated tests for deployment into Openshift and with Podman?